src/main/java/org/opensrp/web/rest/LocationResource.java

```diff
@@ -387,6 +396,42 @@ public void setLocationService(PhysicalLocationService locationService) {
 		return new ResponseEntity<>(identifiers, RestUtils.getJSONUTF8Headers(), HttpStatus.OK);
 
 	}
+	
+	@RequestMapping(value = "/search-location", method = RequestMethod.GET, produces = { MediaType.APPLICATION_JSON_VALUE })
```

Comment:
since the root of the API is `/rest/location` maybe be rename this to '/search-by-tag'